packages/utils/package.json

```diff
@@ -4,6 +4,7 @@
     "version": "1.0.0",
     "main": "src/index.ts",
     "dependencies": {
+        "bowser": "^2.9.0",
```

Comment:
why is this change needed

Reply:
This lib was added as a dependency in account package. But it was not used in account package. But it was used in utils package - `getLoginHistoryFormattedData` Only this function was used in account package. So moved the dependency to utils